modules/bridge/src/main/ts/ephox/bridge/api/Dialog.ts

```diff
@@ -124,10 +124,8 @@ export {
   IframeSpec,
   createIframe,
 
-  ImageTools,
-  ImageToolsSpec,
-  ImageToolsState,
-  createImageTools,
+  ImagePreview as ImagePreview,
+  ImagePreviewSpec as ImagePanelSpec,
```

Comment:
```suggestion
  ImagePreview,
  ImagePreviewSpec,
```
Ahh good old IDEs not cleaning up when renaming :joy:

Reply:
ah right I did the renames before a text search for `imagepanel` so this wasn't picked up.